authentication-request-handler/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,12 @@
+log4j.appender.Stdout=org.apache.log4j.ConsoleAppender
```

Comment:
Add license.

Why do we have to have a separate log4j properties file here? Can't we use the main one?

Reply:
Fixed with c907fcc2a908e093c988f8bf86c54ce8e98b72c0